src/Colors.js

```diff
@@ -28,13 +28,30 @@ const COLORS = {
     YELLOW: [255, 255, 0, 255]
 };
 
-const colorValues = Object.values(COLORS);
+const colorKeys = Object.keys(COLORS);
 
-const randomColor = function() {
-    const colorIndex = Math.floor(Math.random() * colorValues.length);    
-    return colorValues[colorIndex];
+const randomColor = function(exclusionList=[]) {
+    const filteredList = exclusionList.length ? filterList(colorKeys, exclusionList) : colorKeys;
+    const colorIndex = Math.floor(Math.random() * filteredList.length);
+    return COLORS[filteredList[colorIndex]];
 };
 
-COLORS.randomColor = randomColor;
+const filterList = (colorKeys, exclusionList) => {
+    if (colorKeys.length === exclusionList.length) {
+        return ["WHITE"];
```

Comment:
Maybe this should return nothing? If someone says give me this list but filter our everything in the list, I'm not sure if we should assume any defaults

Reply:
I guess I'm saying it implies some bug in their logic (or just insanity) so we should make it clear that they get nothing